.eslintrc.json

```diff
@@ -1,124 +1,113 @@
 {
-    "env": {
-        "browser": true,
-        "node": true,
-        "es6": true
-    },
-    "plugins": [
-        "@typescript-eslint",
-        "jsdoc",
-        "react",
-        "prettier",
-        "patternfly-react"
+  "env": {
+    "browser": true,
+    "node": true,
+    "es6": true
+  },
+  "plugins": ["@typescript-eslint", "jsdoc", "react", "prettier", "patternfly-react", "compat"],
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:react/recommended",
+    "plugin:jsdoc/recommended",
+    "plugin:compat/recommended"
```

Comment:
Is it really necessary to add this to the eslint config? Most teams use `babel` or `tsc` or other build tools that take care of shims or polyfills for them.